edx_toggles/toggles/__init__.py

```diff
@@ -3,30 +3,8 @@
 """
 from .internal.setting_toggle import SettingDictToggle, SettingToggle
 from .internal.waffle.flag import NonNamespacedWaffleFlag, WaffleFlag
-from .internal.waffle.legacy import WaffleFlag as _LegacyWaffleFlag
-from .internal.waffle.legacy import WaffleFlagNamespace as _LegacyWaffleFlagNamespace
-from .internal.waffle.legacy import WaffleSwitch as _LegacyWaffleSwitch
-from .internal.waffle.legacy import WaffleSwitchNamespace as _LegacyWaffleSwitchNamespace
+from .internal.waffle.legacy import WaffleFlag as LegacyWaffleFlag
+from .internal.waffle.legacy import WaffleFlagNamespace as LegacyWaffleFlagNamespace
+from .internal.waffle.legacy import WaffleSwitch as LegacyWaffleSwitch
+from .internal.waffle.legacy import WaffleSwitchNamespace as LegacyWaffleSwitchNamespace
 from .internal.waffle.switch import NonNamespacedWaffleSwitch, WaffleSwitch
-
-
-# We create new classes instead of simply importing the legacy classes
-# for better monitoring of legacy class usage.
-class LegacyWaffleFlag(_LegacyWaffleFlag):
-    def _get_legacy_custom_attribute_name(self):
-        return "deprecated_compatible_legacy_waffle_class"
```

Comment:
1. Probably because it is late, it took me a while to understand what was going on here.  I think you are renaming `deprecated_compatible_legacy_waffle_class` to `deprecated_legacy_waffle_class`, which I will need to change in my monitoring.  Is that correct?  Please update the changelog.  Thanks.
2. Could we simply rename `WaffleSwitch` to `LegacyWaffleSwitch`, as well as the others, inside `.internal.waffle.legacy`? This would make it more obvious everywhere, including in the custom attributes, that these are the Legacy classes.  That is what having these class names in this file had provided.

Reply:
> Probably because it is late, it took me a while to understand what was going on here. I think you are renaming deprecated_compatible_legacy_waffle_class to deprecated_legacy_waffle_class, which I will need to change in my monitoring. Is that correct? Please update the changelog. Thanks.

I believe this is correct -- but I'm still very fuzzy about custom attributes and how they are used on your side, so I'm not sure what is the final impact.

> Could we simply rename WaffleSwitch to LegacyWaffleSwitch, as well as the others, inside .internal.waffle.legacy?

Yes. I'll do that.

Reply:
Great.  To clarify, [in this code](https://github.com/edx/edx-toggles/blob/e654795315bf05a6053534da5b10f672cffb55d0/edx_toggles/toggles/internal/waffle/legacy.py#L58-L63), we refer to the class name in the value, and it would be nice if it contain `Legacy`.  The requested change will make that happen.